src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -454,10 +454,10 @@
     <value>Toggle command palette</value>
   </data>
   <data name="SuggestionsCommandHistoryCommandKey" xml:space="preserve">
-    <value>Recent commands</value>
+    <value>Recent commands...</value>
   </data>
   <data name="SuggestionsCommandKey" xml:space="preserve">
-    <value>Open suggestions</value>
+    <value>Open suggestions...</value>

```

Comment:
Reverting this one. We use it [here](https://github.com/microsoft/terminal/blob/60ac45c2398e3d8f04680559188526b75b29fde8/src/cascadia/TerminalSettingsModel/ActionArgs.cpp#L668) so it'd look weird with the `...`. Besides, we actually are opening the suggestions. We're not necessarily asking for more information.